tgui/packages/tgui/interfaces/AntagInfoThief.tsx

```diff
@@ -0,0 +1,78 @@
+import { BooleanLike } from 'common/react';
```

Comment:
whyd oes it need it's own UI?

Reply:
I made a generic UI because not every antag ui was complete yet, many antagonists have UI to be a unique introduction into the role that stores objectives, round info, and tips on how to play. I want to eventually get rid of generic UI entirely to achieve this goal

Reply:
I will also note that the most expensive thing you can do is add a new theme, which this doesn't do ;)